features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Shared/LeftMenuItem.jsx

```diff
@@ -122,78 +149,84 @@ function LeftMenuItem(props) {
         activeBackground = leftMenu.leftMenuActiveSubmenu;
     }
     return (
-        <Link
-            className={classNames(
-                classes.leftLInk,
+        <BootstrapTooltip title={props.text} placement="right">
+            <Link
+                className={classNames(
+                    classes.leftLInk,
+                    {
+                        [classes.leftLink_IconLeft]: leftMenu === 'icon left',
+                        [classes.submenu]: submenu,
+                    },
+                    'leftLInk',
+                )}
+                to={to}
+                style={{ backgroundColor: activeBackground }}
+                title={props.text}
```

Comment:
IMO it's better to do props destructuring at the top level, Should have caught by eslint :/

Reply:
done